browser-test/src/support/admin_translations.ts

```diff
@@ -0,0 +1,19 @@
+import { Page } from 'playwright'
+
+export class AdminTranslations {
+  public page!: Page
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async selectLanguage(language: string) {
+    await this.page.click(`.language-link:has-text("${language}")`);

```

Comment:
Suggestion (blocking): please prefer `:text("search-text")` over `:has-text("search-text")` as the latter returns any parents that contain an element that has the matching text, while the former returns the immediate element that contains the matching text.

Reply:
Ah that's good to know, thank you!

Reply:
Done!